lib/checks/aria/aria-required-children-evaluate.js

```diff
@@ -16,18 +17,24 @@ function getOwnedRoles(virtualNode, required) {
     const ownedElement = ownedElements[i];
     const role = getRole(ownedElement, { noPresentational: true });
 
+    const hasGlobalAria = getGlobalAriaAttrs().some(attr =>
+      ownedElement.hasAttr(attr)
+    );
+    const includedInAccessibilityTree =
+      hasGlobalAria || isFocusable(ownedElement);
```

Comment:
Wrong name. I think what you're testing for is presentational role conflict, not if it's in the tree.